src/relay/op/nn/nn.h

```diff
@@ -88,6 +90,29 @@ bool DenseRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
   return true;
 }
 
+template <typename AttrType>
```

Comment:
should we keep this function in `nn.cc`?

Reply:
I keep ```DensePackRel``` here since ```DenseRel``` is in ```nn.h```.